ca_on_toronto/__init__.py

```diff
@@ -1,22 +1,49 @@
+from utils import CanadianJurisdiction
 from opencivicdata.divisions import Division
 from pupa.scrape import Organization
 
-from utils import CanadianJurisdiction
-
 
 class Toronto(CanadianJurisdiction):
-    classification = "legislature"
-    division_id = "ocd-division/country:ca/csd:3520005"
-    division_name = "Toronto"
-    name = "Toronto City Council"
-    url = "http://www.toronto.ca"
+    classification = 'executive'  # just to avoid clash
+    division_id = 'ocd-division/country:ca/csd:3520005'
```

Comment:
Please leave the double quotes. I added `black` as a pre-commit hook, so that we don't need to spend time on style changes in the future.